io.github.hydrusnetwork.hydrus.yml

```diff
@@ -26,6 +26,7 @@ finish-args:
   - --share=network
   - --device=dri
   - --filesystem=host
+  - --own-name=org.kde.*
```

Comment:
Someone needs to check if it is actually working. Without this the app says tray icon isn't supported, but once I add this I can't select the option to enable tray icon (the checkbox doesn't work).

Reply:
Note to self. Replace with `--talk-name=org.kde.StatusNotifierWatcher`

Reply:
> Someone needs to check if it is actually working. Without this the app says tray icon isn't supported, but once I add this I can't select the option to enable tray icon (the checkbox doesn't work). Maybe it is a theme issue?

Ok there is a toggle in help>advanced mode that needs to be enabled to show tray option. Works after that.

Reply:
Done